website/snippets/_cloud-environments-info.md

```diff
@@ -42,3 +42,48 @@ By default, all environments will use the default branch in your repository (usu
 - **Deployment:** determines the branch is cloned during job executions for each environment.
 
 For more info, check out this [FAQ page on this topic](/faqs/Environments/custom-branch-settings)!
+
+
+### Extended Attributes (Beta)
+
+:::important This feature is currently in beta
+
+Extended Attributes allows users to enter YAML attributes in their Environment settings. It's currently in beta and if you're interested in getting access to the beta, please contact us -- HOW CAN USES TRY THIS BETA? IF IT'S FEATURE FLAG, HOW DO THEY JOIN THE BETA? DO WE NEED TO ADD A FORM LINK? OR EMAIL?
```

Comment:
@Fleid how can users access or join the beta? do we want this 'join beta' sentence/callout?